docs/use-the-network/console/data-credits.mdx

```diff
@@ -0,0 +1,41 @@
+---
+id: data-credits
+hide_title: true
+sidebar_label: Data Credits
+slug: /use-the-network/console/data-credits
+---
+
+# Data Credits
+
+Data Credits (DCs) are central to the Helium Blockchain and LongFi. Read about
+how they fit into our Tokenomics
+[here](/blockchain/helium-token#data-credits-and-burn-and-mint-economics).
+
```

Comment:
:::info 

You can use the [Helium Data Credit Calculator](https://www.helium.com/use) to understand exactly how much it will cost to send device data on the Network.

:::